workflow/rules/vardict.smk

```diff
@@ -0,0 +1,33 @@
+# vim: syntax=python tabstop=4 expandtab
+# coding: utf-8
+
+__author__ = "Jonas Almlöf"
+__copyright__ = "Copyright 2021, Jonas Almlöf"
+__email__ = "jonas.almlof@scilifelab.uu.se"
+__license__ = "GPL-3"
+
+
+rule vardict:
+    input:
+        bam="alignment/extract/{sample}_{type}_{chr}.bam",
+        reference=config["reference"]["fasta"],
+        regions="misc/bed_split/{sample}_{type}_{chr}.bed",
+    output:
+        vcf=temp("snv_indels/vardict/{sample}_{type}_{chr}.unfilt.vcf.gz"),
+    params:
+        extra=config.get("vardict", {}).get("extra", ""),
+        bed_columns=config.get("vardict", {}).get("bed_columns", ""),
```

Comment:
Here we need to set a column default, the current version remove all information about which columns can be expected in the bed file.

default suggestion: "-c 1 -S 2 -E 3 -g 4"

Reply:
```suggestion
        bed_columns=config.get("vardict", {}).get("bed_columns", "-c 1 -S 2 -E 3"),
```